### PR TITLE
contribex: Add teams for sigs-github-actions repo

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
Ref: https://github.com/kubernetes/org/issues/4218

/assign @nikhita

### CODE REVIEW

File: config/kubernetes-sigs/sig-contributor-experience/teams.yaml [hunk 1]
```diff
@@ -61,6 +61,23 @@ teams:
...
     privacy: closed
+  sigs-github-actions-admins:
+    description: admin access to the sigs-github-actions repo
+    maintainers:
+    - cblecker
+    - idvoretskyi
+    - MadhavJivrajani
+    - mrbobbytables
+    - nikhita
+    - palnabarun
```
Comment (palnabarun) [inline config/kubernetes-sigs/sig-contributor-experience/teams.yaml:72]:
```suggestion
    - palnabarun
    privacy: closed
```

File: config/kubernetes-sigs/sig-contributor-experience/teams.yaml [hunk 2]
```diff
@@ -61,6 +61,23 @@ teams:
...
+  sigs-github-actions-maintainers:
+    description: write access to the sigs-github-actions repo
+    maintainers:
+    - cblecker
+    - MadhavJivrajani
+    - nikhita
+    members:
+    - Priyankasaggu11929
```
Comment (palnabarun) [inline config/kubernetes-sigs/sig-contributor-experience/teams.yaml:80]:
```suggestion
    - Priyankasaggu11929
    privacy:closed
```

**Comment by MadhavJivrajani** [comment]
Done 🙏🏼

**Review by palnabarun — APPROVED** [review]
/lgtm
/approve

**Pull request merged**